app/notification_push.go

```diff
@@ -78,13 +78,28 @@ func (a *App) sendPushNotificationToAllSessions(msg *model.PushNotification, use
 		return err
 	}
 
+	notification, parseError := model.PushNotificationFromJson(strings.NewReader(msg.ToJson()))
+	if notification == nil || parseError != nil {
```

Comment:
AFAIU, there can never be a case where parseError is not nil and notification is also not nil. So I think just checking for `parseError != nil` is sufficient.

Reply:
I've check for the input and returned error in case the input is nil. I think is a better approach and a more faithful error message in case it happens